src/web/components/Card/components/AvatarSize.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
```

Comment:
As both avatarContainer and avatarSize are in the `Card` folder, do they need to be separated? I can see them being separate if the container is used elsewhere and the size is somewhere else but I wonder whether they're in the right place/could be combined/could have props passed to size so we can use everywhere avatar is?

Reply:
As discussed irl, it's good to keep `Avatar` simple and use wrappers to add sizing and margins. In this case we have a Card specific wrapper.

I merged `ArticleSize` into `ArticleContainer`.